providers/implementations/signature/rsa_sig.c

```diff
@@ -765,9 +783,9 @@ static int rsa_sign(void *vprsactx, unsigned char *sig, size_t *siglen,
 static int rsa_verify_recover_init(void *vprsactx, void *vrsa,
                                    const OSSL_PARAM params[])
 {
-    if (!ossl_prov_is_running())
-        return 0;
-    return rsa_signverify_init(vprsactx, vrsa, params,
+    PROV_RSA_CTX *prsactx = (PROV_RSA_CTX *)vprsactx;
+
+    return rsa_signverify_init(prsactx, vrsa, params,
```

Comment:
This is IMO also unapproved operation -> the same indicator should apply.

Reply:
Agreed